recOrder/plugin/main_widget.py

```diff
@@ -190,6 +194,24 @@ def __init__(self, napari_viewer: Viewer):
         self.ui.qbutton_acq_phase_from_bf.clicked[bool].connect(
             self.acq_phase_from_bf
         )
+        self.bf_keywords = [
```

Comment:
This can also be a class attribute if it's never modified in instances.

Reply:
Can you help me understand when/why I might use class attributes? In this case we'll only have a single instance of this class, right?

(This is a question almost entirely from ignorance...I haven't used class attributes much).

Reply:
In this case it does not really matter since nothing is actually private in Python. Here it'll be more like a visual discouragement against modifying what's intended to be a constant. Feel free to leave it this way!

Reply:
Thanks! I understand what's going on now.